testsuite/features/step_definitions/command_steps.rb

```diff
@@ -878,7 +878,9 @@
   node = get_target(host)
   os_version, _os_family = get_os_version(node)
   cmd = 'false'
-  if (os_version.include? '12') || (os_version.include? '15')
+  if (os_version.include? '15') && (host.include? 'proxy')
+    cmd = "mgr-create-bootstrap-repo -c SUMA-41-PROXY-#{arch}"
```

Comment:
I wonder if we bootstrap the same repo name in Uyuni 🤔

Reply:
Don't know.

Apart from that, same as above, we could also start using 42.

Reply:
Yes. For head code, this is a typo. Still we need to distinguish products.
Pinging our proxy master, @lkotek

Reply:
I switched to 4.2.

Thanks for the ping. We can fix the issue here, or in a separate PR, whatever makes more sense for you guys.

Reply:
I will take a look. I have a lot of notes about it from few months ago :-)

Reply:
@srbarrios In case of uyuni we create same bootstrap repo as for Leap 15.2. Proxy specific things needs to be part of activation key, but there is nothing specific to be done at level of bootstrap repo. (Basically uyuni works like SUMA 3.2 and older.)

Reply:
So for Uyuni we will also use `SUMA-41-PROXY-#{arch}` this on the bootstrap repo command ?

Reply:
I will remove the change and let Lukas address this in a separate PR.